server/Dockerfile

```diff
@@ -2,6 +2,17 @@ FROM node:lts
 
 WORKDIR /app
 COPY ./package*.json ./
+
+# This installs Chromium, which is required by puppeteer to run
+RUN apt-get update \
+    && apt-get install -y wget gnupg \
```

Comment:
Does this not get run with npm install?

Reply:
yeah sorry! the comment is actually not that accurate, it installs all the dependencies for chromium instead. i'll fix that